en/10_Contributing/02_Documentation.md

```diff
@@ -93,30 +93,59 @@ yarn lint --fix
+- Avoid using a $ symbol for bash code blocks.
+- Code examples must follow [coding conventions](https://docs.silverstripe.org/en/contributing/coding_conventions/) for that language.
+- Use "App" as the PHP namespace when defining code that is meant to be added at the project level.
+- Include the filename that a code block relates to, do so in a comment in the first line of the code block. Don't place a blank line after this comment, unless the next line is also a comment.
+
+### References to various things
+
+- Add a trailing slash to directory references to distinguish from regular files.
+- [Link to Silverstripe API docs](#links-to-api-documentation) for the first reference to a Silverstripe PHP class on a page. After that, surrond the class name with backticks.
 - Use `example.com` as the domain of sample URLs and `example.org` and `example.net` when additional domains are required. All of these domains are [reserved by the IANA](https://tools.ietf.org/html/rfc2606#section-3).
-- Use appropriate markdown. The use of code blocks, emphasis, lists, and tables make technical writing easier to scan.
-- Never leave spaces at the end of a line.
-- Use an empty line to separate paragraphs.
-- Sentence case your bullet points. If a bullet point is a full sentence then end with a full stop. If it is a sentence fragment or a comma separated list, full stops are not required.
-- All code blocks must have an appropriate syntax highlighting hint. For plain text, use "text".
-- If it's useful to include the filename that a code block relates to, do so in a comment in the first line of the code block.
-  - Don't place a blank line after this comment, unless the next line is also a comment.
-- Code examples should follow our [coding conventions](./coding_conventions).
-- Use `App` when the code requires a vendor name (e.g. as the root for namespaces).
-- Do not put a `$` in front of lines of bash code examples.
-- When referencing directories, always add a trailing slash to avoid confusions with regular files.
-- "Silverstripe CMS" is used to refer to the Silverstripe CMS open source project as a whole.
-- "Silverstripe" is used to refer to the company, where appropriate.
+
+### References to Silverstripe
+
+- Spell "Silverstripe" with an uppercase 'S' at the beginning and a lowercase 's' in the middle.
+- Refer to the company as "Silverstripe Ltd."
+- Use "Silverstripe CMS" to refer the content management system.
+
+### Markdown linting
+
+- Files that are within a directory specified in repository's `.doclintrc` file will be automatically linted in CI using [silverstripe/documentation-lint](https://github.com/silverstripe/documentation-lint/).
+- Many of the linting rules are not mentioned in this writing style guide, though all linting rules will need to be adherred to in order for your documentation pull-request to be merged. The failed linting CI build will clearly highlight what needs to be fixed.
```

Comment:
We definitely don't need the technical details about a `.doclintrc` file - IMO we don't need this section at all but if you want to keep it, make the first point just something like "Documentation will be automatically linted in CI"

Reply:
Done